brozzler/frontier.py

```diff
@@ -30,6 +33,57 @@ class UnexpectedDbResult(Exception):
     pass
 
 
+def filter_claimable_site_ids(
+    active_sites: List[Dict], max_sites_to_claim=1
+) -> List[str]:
+    job_counts = {}
+    claimable_sites = []
+    now = datetime.datetime.now(datetime.timezone.utc)
+
+    for site in active_sites:
+        is_claimable = False
+
+        # If site not claimed and not disclaimed within last 20 seconds
+        if not site["claimed"] and site.get("last_disclaimed", 0) <= (
```

Comment:
Is `"claimed"` guaranteed to be in `site`? I noticed you're guarding against all of the other keys, but not this one.

Reply:
Yes, good to double check that. There is a default of False, and also the input is a result of a query filtering on that field, so it has to exist at this point.